numerics_paper/version2/numerical_chj.tex

```diff
@@ -709,12 +688,12 @@ \subsection{Dam break problem on a dry bed}\label{sec:rp_dry_bed}
 \quad
   u(x,t) =
   \begin{cases}
-    0, &\mbox{ if } x\leq x_A(t), \\
-    \frac{2}{3}\left(\sqrt{gh_l}+\frac{x-x_0}{t}\right), & \mbox{ if } x_A(t) < x\leq x_B(t), \\
-    0, &\mbox{ if } x_B(t)\leq x,
+    0, &\mbox{ if } x\leq x_-(t), \\
+    \frac{2}{3}\left(\sqrt{gh_l}+\frac{x-x_0}{t}\right), & \mbox{ if } x_-(t) < x\leq x_+(t), \\
+    0, &\mbox{ if } x_+(t)\leq x,
   \end{cases}
 \end{align*}
-with $x_A(t)=x_0-t\sqrt{gh_l}$ and $x_B(t)=x_0+2t\sqrt{gh_l}$.
+with $x_\pm(t)=x_0 \pm t\sqrt{gh_l}$.
```

Comment:
Were you aware there is a 2 before t in the definition of x_B? I should double-check the reference.

Reply:
Oops, you're right.  I will put it back.